docs/docs/concepts/overview.md

```diff
@@ -415,7 +415,21 @@ At the moment, Optimus supports BigQuery datastore for 3 types of resources:
 TODO
 
 ## Replay & Backups
-TODO
+A job might need to be re-run (backfill) due to business requirement changes or other various reasons. Optimus provides 
+an easy way to do this using Replay feature. Replay accepts which job and range of date to be updated, validating it, 
+and clearing the tasks in the scheduler, making the tasks available to be picked up by the scheduler and got re-run. 
```

Comment:
picked up by the scheduler and `for` re-run.

Reply:
removed this part as it is mentioned in the next paragraph.